components/core/src/env.rs

```diff
@@ -154,9 +154,9 @@ macro_rules! env_config {
 /// ```
 #[macro_export]
 macro_rules! env_config_duration {
-    ($(#[$attr:meta])* $wrapping_type:ident, $env_var:ident => $from_str_fn:ident, $default_value:expr) => {
+    ($(#[$attr:meta])* $vis:vis $wrapping_type:ident, $env_var:ident => $from_str_fn:ident, $default_value:expr) => {
```

Comment:
:thinking: Why did you end up having to add this?

Reply:
I needed to declare HttpStartupTimeout as `pub` because the `http_startup_timeout` field is public. Thats what the compiler told me to do.